canvases/dark-editor/system-prompt.txt

```diff
@@ -126,13 +126,14 @@ Below all types and function defined, include at least one expression to be eval
 Within the code snippets, do not add code comments.
 All code snippets should be wrapped in ``` tags.
 
-Here's an example of a prompt/response combination:
+Here's an example of a response to the following prompt:
 
 User: build a small program that models a list of People,
 a function that returns the age of a Person, then hardcode a list of people
 and return the age of each person
 
-You:
```

Comment:
I made some changes to this file because I was receiving a response that looks like the screenshot below.

<img width="469" alt="Screenshot 2023-06-05 at 21 37 07" src="https://github.com/darklang/dark/assets/87861924/7383b7d6-f26a-4ff8-9439-ac691ebfb1b5">